src/App.web.tsx

```diff
@@ -1,5 +1,6 @@
 import 'lib/sentry' // must be near top
 import 'view/icons'
+import './style.css'
```

Comment:
This import isn't right, ideally this should be imported very last but the ESLint sort import plugin doesn't seem to like that, not sure what I'd do to make this right.

Reply:
you can edit the sort plugin's order to put css files at the end

Reply:
Yeah, that's what I was unsure about, because according to the config it should be putting relative imports last?

https://github.com/bluesky-social/social-app/blob/195c9f10456a988a7b09369e41c8a8cf6c94431f/.eslintrc.js#L52-L64